docoptTests/DocoptTestCaseParser.swift

```diff
@@ -91,7 +96,13 @@ public struct DocoptTestCaseParser {
             NSLog("Error parsing \(expectedOutput) to JSON: \(error)")
             return nil
         }
-        let expectedOutputJSON: AnyObject? = NSJSONSerialization.JSONObjectWithData(jsonData!, options: .AllowFragments, error: &error)
+        let expectedOutputJSON: AnyObject?
```

Comment:
rather, swift 2.0 auto-handles nserror fields for you behind the scenes, and considers the return of one the same as a thrown exception that must be caught __somewhere__.